src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  position: relative;
+  width: 96px;
+  top: 22px;
+  left: 22px;
+}
+
+.stars__star {
+  display: inline-block;
+  margin-right: 5px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
```

Comment:
It's good to see that you have a separate class for styling stars. However, avoid using the 'margin-right' property on the last child element as it can cause unnecessary spacing. Consider using the ':not(:last-child)' pseudo-class to apply the margin to all except the last child.